web/pgadmin/__init__.py

```diff
@@ -265,11 +265,14 @@ def create_app(app_name=None):
         logger.addHandler(fh)
 
     # Console logging
-    ch = logging.StreamHandler()
-    ch.setLevel(config.CONSOLE_LOG_LEVEL)
-    ch.setFormatter(logging.Formatter(config.CONSOLE_LOG_FORMAT))
-    app.logger.addHandler(ch)
-    logger.addHandler(ch)
+
+    formatter = JsonFormatter(config.CONSOLE_LOG_FORMAT)
```

Comment:
We need both options, plain text and JSON logging. So, keep the original (plain text) as a default and the JSON logging configurable.